README.md

```diff
@@ -42,6 +42,43 @@ EOF
 systemctl restart hyperd
 ```
 
+Setup CNI networking
+
+Follow the steps below in order to setup networking in your pods using the CNI
```

Comment:
Please merge those two into one short line (so as to in same style as other places)